azure-pipelines.yml

```diff
@@ -32,19 +32,32 @@ stages:
     - task: UseDotNet@2
       inputs:
         version: '8.x'
+    - task: SonarQubePrepare@5
+      inputs:
+        SonarQube: 'ofqual-register-api-service-connection'
+        scannerMode: 'MSBuild'
+        projectKey: 'OfqualGovUK_ofqual-register-api_AY4YamDUJEdxy1l8gwpa'
```

Comment:
Please update the projectKey to add it into an environment variable like the Azure Subscription

Reply:
Done Syed